### ISSUE TITLE
Dependency check: Ignore patch versions

### ISSUE THREAD

**Issue by KristjanESPERANTO** [MagicMirrorOrg/MagicMirror-3rd-Party-Modules#12]
It doesn't make much sense to list patch versions in the dependency check. We should filter these out.

It might also be worth considering doing this with minor versions as well.

Here is an actual example of a patch version change:

```
1. Information: There are updates for 1 dependencie(s):
     esbuild ^0.20.0 → ^0.20.1
```

**Note.** Semantic versioning should normally be structured like this: MAJOR.MINOR.PATCH

**Comment by dathbe** [comment]
Aren't patch versions often MORE important than major versions?  If I understand semantic versioning correctly, the patch versions are going to plug security holes (potentially), which would be important to catch and update.  Major versions are going to have substantive changes that aren't necessary if the module is working, and potentially could break things.

**Comment by KristjanESPERANTO** [comment]
You are absolutely right about the patch versions. It wouldn't really be wise to filter them out. I don't know why I thought that was a good idea when I created the issue 😅

Since some major and minor releases also fix security issues, we can't filter them out for the same reason.

Once again, thank you for your feedback! I hereby close the issue. If you have any further thoughts on this subject, do not hesitate to comment.
